pkg/config/config.go

```diff
@@ -19,7 +19,7 @@ type Config struct {
 	Password         string `json:"password,omitempty"`
 	Username         string `json:"username,omitempty"`
 	Port             uint16 `json:"port"`
-	ContainerManager string `json:"container-manager,omitempty"`
+	ContainerManager string `json:"container_manager,omitempty"`
```

Comment:
boooo :-1:

Reply:
well that makes ruby templates significantly easier change seems better. also there is a mismatch with deployer_path and container_manager.

Reply:
~~~AND YET `container-manager` existed first~~~

but really it's just harder to type `_` rather than `-`.

I cannot help that ruby sucks, but okay.